test/Project.toml

```diff
@@ -1,4 +1,5 @@
 [deps]
+Cloudy = "9e3b23bb-e7cc-4b94-886c-65de2234ba87"
```

Comment:
For testing purposes, we should avoid adding the same package name in the test folder's Project.toml (so that GitHub uses the PR code instead of registered package while testing (or something like this according to Charlie)). For testing locally you could use `TestEnv` which includes the package and other tests requirements activated in a temporary folder.

Reply:
Oh awesome, thank you! I was previously following the ClimaAtmos protocol of adding the package and then writing `dev .` to make sure it used the local version. But this workflow seems better.

Reply:
Removed this from the Project.toml